en/app.ftl

```diff
@@ -92,14 +92,14 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
 hero-image-copy-protect-html = … so you can <strong>protect your real email</strong> from tracking and spam.
 hero-image-copy-control-html = Now <em>you’re in control</em> of what’s arriving to your inbox!
 
-# Home Page Version A
+## Home Page Version A
 
-# Hero Section
+## Hero Section
```

Comment:
@peiying2  I followed the document that you sent me, let me know if this is the right way to go about writing comments.

Reply:
> @peiying2 I followed the document that you sent me, let me know if this is the right way to go about writing comments.

Check the doc's examples: 
- Double-hash comments (##) are always standalone, meaning there should be line space between the comment and the start of the strings
- For string specific # comment, it is immediately above the string. 

Please go through all the comments and fix those you have missed.